schemas/quality_performance_score.yaml

```diff
@@ -0,0 +1,44 @@
+---
+quality_performance_score:
```

Comment:
will the response always have `data` and `error` properties? `data` isn't defined in the schema here